content/doc/book/pipeline/docker.adoc

```diff
@@ -372,6 +372,38 @@ node {
 }
 ----
 
+The `build()` method builds the `Dockerfile` in the current directory by 
+default. This can be overridden by providing a second argument with the path of
+a directory containing a `Dockerfile`, for example:
+
+[source,groovy]
+----
+node {
+    checkout scm
+    def testImage = docker.build("test-image", "./dockerfiles/test")
+
+    testImage.inside {
+        sh 'make test'
+    }
+}
+----
+
+It is possible to pass arguments that 
+link:https://docs.docker.com/engine/reference/commandline/build/[docker build]
+accepts via the second argument of the `build()` method. When this argument is
+supplied a path is required as well.
+
+This is another way to override the default `Dockerfile`, by passing the `-f`
+flag, for example:
+
+[source,groovy]
+----
+node {
+    checkout scm
+    def dockerfile = 'Dockerfile.test'
+    def customImage = docker.build("my-image:${env.BUILD_ID}", "-f ${dockerfile} .")
```

Comment:
How about using a path similar to previous examples?  Instead of `.`, use `./dockerfiles`.